### PR TITLE
Fixes #1212 - Remove code to support SAML

### LINKED ISSUE #1212: Remove the SAML code
Since Myla initially stated with SAML for authentication and now we moved out of that with LTI. we should totally remove the code associated with it.

### PR DESCRIPTION
**Pull request by jonespm**
I tested both with LTI and non and it looks fine. I don't see any more SAML reference.

### CODE REVIEW

**Review by zqian — COMMENTED** [review]
The code runs fine. 

Just a couple of files to remove the saml support:
.dockerignore
.env.sample
.gitignore

**Review by ssciolla — COMMENTED** [review]
As @zqian said, I found a couple references to saml in `.gitignore`, `.dockerignore`, and `.env.sample`, if you want to remove those.

Generally speaking, this looks like what I would have done. I think we need to retain the `LOGOUT_REDIRECT_URL` AND `LOGIN_REDIRECT_URL` to make the default login work well.

I'm investigating a couple other issues. I'm seeing weird behavior when clicking around the admin site and login/logout; things seem to be lagging, but then work if I try the action again. Quite possibly not related to this PR, but I'm going to run off of master to determine.

**Comment by ssciolla** [comment]
> I'm investigating a couple other issues. I'm seeing weird behavior when clicking around the admin site and login/logout; things seem to be lagging, but then work if I try the action again. Quite possibly not related to this PR, but I'm going to run off of master to determine.

Well, the stalling in the admin site doesn't seem to be limited to the PR, so I guess a separate issue. Maybe it was introduced by the dependency update? I don't remember seeing this. Has anyone else experienced this? Basically, sometimes I click a link to login or logout, or go somewhere on the admin site, and it sits there loading until I hit the link again, and then it works. Very strange.

File: dashboard/settings.py [hunk 1]
```diff
@@ -88,17 +87,13 @@
...
-STUDENT_DASHBOARD_SAML = ENV.get('STUDENT_DASHBOARD_SAML', False)
 
 # Defaults for DEBUGPY
 DEBUGPY_ENABLE = ENV.get("DEBUGPY_ENABLE", False)
 DEBUGPY_REMOTE_ADDRESS = ENV.get("DEBUGPY_REMOTE_ADDRESS", "0.0.0.0")
 DEBUGPY_REMOTE_PORT = ENV.get("DEBUGPY_REMOTE_PORT", 3000)
 DEBUGPY_WAIT_FOR_ATTACH = ENV.get("DEBUGPY_WAIT_FOR_ATTACH", False)
 
-LOGIN_REDIRECT_URL = ENV.get('DJANGO_LOGIN_REDIRECT_URL', '/')
-LOGOUT_REDIRECT_URL = ENV.get('DJANGO_LOGOUT_REDIRECT_URL', '/')
```
[the remark this answers is not in the document]
Comment (jonespm) [reply dashboard/settings.py:100]: Yeah, I saw the **/accounts/profile** come up locally and was confused. I guess easier to just leave these right in there. Thanks!

File: config/env_sample.json [hunk 2]
```diff
@@ -135,6 +135,11 @@
     "/*  Set this to true to enable the backend login/logout URLs.": "*/",
     "/*  This is enabled by default if no external auth is enabled.": "*/",
     "ENABLE_BACKEND_LOGIN": false,
+    "/*  Login Redirect, default is /, probably don't need to change.": "*/",
+    "DJANGO_LOGIN_REDIRECT_URL": "/",
+    "/*  Logout redirect (If SAML enabled)": "*/",
+    "/*  This URL could be configured to the logout, but the default should work fine as LTI doesn't currentlyhave logout.": "*/",
```
Comment (ssciolla) [inline config/env_sample.json:141]:
```suggestion
    "/*  This URL could be configured to the logout, but the default should work fine as LTI doesn't currently have logout.": "*/",
```

File: dashboard/settings.py [hunk 3]
```diff
@@ -94,6 +94,8 @@
 DEBUGPY_REMOTE_PORT = ENV.get("DEBUGPY_REMOTE_PORT", 3000)
 DEBUGPY_WAIT_FOR_ATTACH = ENV.get("DEBUGPY_WAIT_FOR_ATTACH", False)
 
+LOGIN_REDIRECT_URL = ENV.get('DJANGO_LOGIN_REDIRECT_URL', '/')
+LOGOUT_REDIRECT_URL = ENV.get('DJANGO_LOGOUT_REDIRECT_URL', '/')
 # Application definition
```
Comment (ssciolla) [inline dashboard/settings.py:99]:
```suggestion

# Application definition
```

**Review by ssciolla — APPROVED** [review]
LGTM, my (moderate to significant) concerns were resolved.

File: dashboard/settings.py [hunk 4]
```diff
@@ -42,7 +42,6 @@
 LOGIN_URL = '/accounts/login/'
 # Note the absence of the trailing slash.
 # In local development, a slash will be appended by Django during logout.
-# However, the SAML inplementation breaks on logout if a slash is used here.
 LOGOUT_URL = '/accounts/logout'
```
[the remark this answers is not in the document]
Comment (jonespm) [reply dashboard/settings.py:45]: It seems like it doesn't work if I just change it to a slash. Local logout isn't working. 

This setting is being used by utils.py to generate a logout link.

[the remark this answers is not in the document]
Comment (ssciolla) [reply dashboard/settings.py:45]: Sorry, @jonespm, I was saying change it to `/accounts/logout/`, to be transparent about the append slash logic.

[the remark this answers is not in the document]
Comment (jonespm) [reply dashboard/settings.py:45]: Oh okay, yeah can do that!

[the remark this answers is not in the document]
Comment (jonespm) [reply dashboard/settings.py:45]:
```suggestion
LOGOUT_URL = '/accounts/logout/'
```

**Pull request merged**